lib/internal/modules/cjs/loader.js

```diff
@@ -41,9 +41,10 @@ const {
   stripBOM,
   stripShebang
 } = require('internal/modules/cjs/helpers');
-const preserveSymlinks = !!process.binding('config').preserveSymlinks;
-const preserveSymlinksMain = !!process.binding('config').preserveSymlinksMain;
-const experimentalModules = !!process.binding('config').experimentalModules;
+const options = internalBinding('options');
+const preserveSymlinks = !!options.getOptions('--preserve-symlinks');
```

Comment:
there's no need for the `!!` here, as `options.getOptions` already returns true/false

Reply:
I've removed the `!!` in a second commit. I wasn't sure whether I should or not, as they were there for the old code. Thanks!